README.md

```diff
@@ -1,6 +1,653 @@
-Active Directory Plugin
+Active Directory plugin for Jenkins
+===================================
 
-See wiki at: https://wiki.jenkins-ci.org/display/JENKINS/Active+Directory+plugin 
+[![Build Status](https://ci.jenkins.io/job/Plugins/job/active-directory-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/active-directory-plugin/job/master/)
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/active-directory.svg)](https://plugins.jenkins.io/active-directory/)
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/active-directory.svg?label=changelog)](https://github.com/jenkinsci/active-directory-plugin/releases/latest/)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/active-directory.svg?color=blue)](https://plugins.jenkins.io/active-directory/)
+
+| Plugin Information                                                                                            |
+|---------------------------------------------------------------------------------------------------------------|
+| View Active Directory [on the plugin site](https://plugins.jenkins.io/active-directory) for more information. |
+
+Older versions of this plugin may not be safe to use. Please review the following warnings before using an older version:
+
+-   [Improper certificate validation](https://jenkins.io/security/advisory/2019-01-28/#SECURITY-859)
+-   [Man-in-the-middle vulnerability due to missing certificate check](https://jenkins.io/security/advisory/2017-03-20/)
```

Comment:
No need to link from the plugins site to itself and no need to describe security issues.  They are handled by the plugins site generator.
```suggestion
```

Reply:
Removed the link to plugin and security